styles/Stars.module.css

```diff
@@ -8,8 +8,10 @@
   overflow: unset;
   margin-right: 15px;
 }
-.star:hover {
-  opacity: 0.5;
+.star:hover path {
+  cursor: pointer;
+  fill: #ffd600;
+  opacity: 0.8;
```

Comment:
As outlined above, I would use the classNames to target this! No `path` required